tasks/test.rake

```diff
@@ -7,7 +7,6 @@ task :setup, :parallel do |t, opts|
     system 'mkdir -p spec/rails'
     args = %w[
       -m\ spec/support/rails_template.rb
```

Comment:
This line should be `-m spec/support/rails_template_with_data.rb` like in `script/local`